docs-src/cloud/support-service-guarantees.md

```diff
@@ -0,0 +1,19 @@
+---
+id: support-service-guarantees
+title: What are the services offered by Temporal Cloud?
+sidebar_label: Services
+description: We offer four services to Temporal Cloud customers.
+tags:
+  - temporal cloud
+  - support
+---
+
+We offer four services to Temporal Cloud customers. For access to any of these services, please [create a support ticket](/cloud/support) or contact your dedicated account manager.
+
+- **Technical onboarding**: Every Temporal Cloud customer is offered a dedicated onboarding session led by a Developer Success Engineer.
```

Comment:
```suggestion
- **Technical onboarding:** Every Temporal Cloud customer is offered a dedicated onboarding session led by a Developer Success Engineer.
```
Per https://developers.google.com/style/colons#bold-text-preceding-colon (here and three following instances).

Reply:
Good call out.
I have a PR to fix other instances here:
https://github.com/temporalio/documentation/pull/2414